youtube_dl/extractor/indavideo.py

```diff
@@ -51,16 +51,9 @@ def _real_extract(self, url):
             video_urls.append(video_file)
         video_urls = list(set(video_urls))
 
-        video_prefix = video_urls[0].rsplit('/', 1)[0]
-
-        for flv_file in video.get('flv_files', []):
-            flv_url = '%s/%s' % (video_prefix, flv_file)
-            if flv_url not in video_urls:
-                video_urls.append(flv_url)
-
```

Comment:
Oh, I see, was that code added later? I don't remember testing with HD.
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.